docs/tapiro/installation.md

```diff
@@ -0,0 +1,89 @@
+---
+id: installation
+title: Installation
+---
+
+`tapiro` can be installed as an Sbt plugin.
+
+`sbt-tapiro` is an Sbt plugin that uses `tapiro` to generate http/json routes parsing scala traits definitions.
+
+## Installation
+
+To start using `sbt-tapiro` simply add this line in `project/plugins.sbt`
+
+```scala
+addSbtPlugin("io.buildo" %% "sbt-tapiro" % "@SBT_TAPIRO_STABLE_VERSION@")
+```
+
+### Snapshot releases
+
+We publish a snapshot version on every merge on master.
+
+The latest snapshot version is `@SBT_TAPIRO_SNAPSHOT_VERSION@` and you can use
+it to try the latest unreleased features. For example:
+
+```scala
+addSbtPlugin("io.buildo" %% "sbt-tapiro" % "@SBT_TAPIRO_SNAPSHOT_VERSION@")
+resolvers += Resolver.sonatypeRepo("snapshots")
+```
+
+## Plugin
+
+To use the code generator, you need to add this to your `build.sbt`.
+
+```scala
+import _root_.io.buildo.tapiro.Server
```

Comment:
@gabro do you have a better way to do this?

Reply:
you can add an `autoImport` object to the sbt auto plugin so that this value is already available in scope. I would do something like

```scala
  object autoImport {
    lazy val TapiroServer = io.buildo.tapiro.Server
  }
```

See a reference here https://www.scala-sbt.org/1.x/docs/Plugins.html

Reply:
https://github.com/buildo/retro/issues/174